modules/appeals_api/config/shared_schemas/non_blank_string.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "description": "JSON Schema for VA Decision Review Forms: 'nonBlankString'",
+  "$ref": "#/definitions/nonBlankString",
+  "definitions": {
+    "nonBlankString": {
+      "type": "string",
+      "pattern": "[^ \\f\\n\\r\\t\\v\\u00a0\\u1680\\u2000-\\u200a\\u2028\\u2029\\u202f\\u205f\\u3000\\ufeff]",
+      "$comment": "The pattern used ensures that a string has at least one non-whitespace character. The pattern comes from JavaScript's \\s character class. \"\\s Matches a single white space character, including space, tab, form feed, line feed, and other Unicode spaces. Equivalent to [ \\f\\n\\r\\t\\v\\u00a0\\u1680\\u2000-\\u200a\\u2028\\u2029\\u202f\\u205f\\u3000\\ufeff].\": https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes  We are using simple character classes at JSON Schema's recommendation: https://tools.ietf.org/html/draft-handrews-json-schema-validation-01#section-4.3"
```

Comment:
Can we simplify this wording somehow? I've never been a fan of how verbose it is, and we don't go into nearly as much detail with any other `$comment`. I'd honestly be okay with removing it outright, since we strip `$comment`s from our documentation anyway...